src/test/java/xades4j/production/OtherSignerTests.java

```diff
@@ -112,7 +110,7 @@ static class MyResolverSpi extends ResourceResolverSpi
         @Override
         public XMLSignatureInput engineResolveURI(ResourceResolverContext context) throws ResourceResolverException
         {
-            XMLSignatureInput input = new XMLSignatureInput(context.attr.getValue().getBytes());
+            XMLSignatureInput input = new XMLSignatureStreamInput(new ByteArrayInputStream(context.attr.getValue().getBytes()));

```

Comment:
As mentioned in another comment, please revert the Apache Santuario update.

Reply:
yes I'll do this.